src/main/res/values/strings.xml

```diff
@@ -1303,17 +1303,17 @@
 	</plurals>
 
 	<!-- 2020-09-19 -->
-	<string name="action_save_image_success_no_path">Image saved successfully</string>
+	<string name="action_save_image_success_no_path">Image saved</string>
```

Comment:
Preferred original

Reply:
Is an image ever saved unsuccessfully?

Reply:
In general I don't subscribe to the idea that cutting out words from status messages is a goal unto itself, that always makes things more readable. Redundancy is a natural part of language, and often improves clarity.

I don't really mind one way or another in this particular case, I think the original was fine as we should just leave it as-is.

Reply:
> cutting out words from status messages is a goal unto itself, that always makes things more readable.

I agree, not always though. It can't be both redundant and more clear. While fewer words aren't always better, all redundant words can and should be cut.